static/js/containers/ProfilePage.js

```diff
@@ -40,9 +39,7 @@ class ProfilePage extends React.Component {
       dispatch(startProfileEdit());
     }
     return dispatch(validateProfile(profile, requiredFields, messages)).then(() => {
-      dispatch(saveProfile(SETTINGS.username, profile)).then(() => {
-        dispatch(clearProfileEdit());
-      });
+      dispatch(saveProfile(SETTINGS.username, profile));
```

Comment:
@aliceriot it is navigating user to ``personal tab`` before it clears edit profile. 
Thats why when user lands on ``personal profile tab`` all the error messages get clear.